### PR TITLE
note need for Graphviz

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,19 +9,22 @@ Using these abstractions, Pekko also provides libraries for persistence, streams
...
---------------------
+## Building from Source
 
+### Prerequisites
 - Make sure you have installed a Java Development Kit (JDK) version 11 or later.
 - Make sure you have [sbt](https://www.scala-sbt.org/) installed and using this JDK.
 - Make sure you also have Java 8 installed (JDK or JRE).
-- Use git to clone the [repo](https://github.com/apache/incubator-pekko) or download a source release from https://pekko.apache.org
-- Open a command window and change directory to the directory where you installed the source
+- [Graphvis](https://graphviz.gitlab.io/download/) is needed for the scaladoc generation build task, which is part of the release.
```
Comment (raboof) [inline README.md:22]:
```suggestion
- [Graphviz](https://graphviz.gitlab.io/download/) is needed for the scaladoc generation build task, which is part of the release.
```

File: README.md [hunk 2]
```diff
@@ -35,11 +37,10 @@ Building from Source
      - Requires Java 11 minimum
      - `sbt docs/paradoxBrowse` does the same but will open the docs in your browser when complete
      - the `index.html` file will appear in `target/paradox/site/main/`
-- `sbt unidoc` will build the Javadocs for all the modules and load them to one place
+- `sbt unidoc` will build the Javadocs for all the modules and load them to one place (may require Graphvis, see Prerequisites above)
```
Comment (raboof) [inline README.md:40]:
```suggestion
- `sbt unidoc` will build the Javadocs for all the modules and load them to one place (may require Graphviz, see Prerequisites above)
```

File: README.md [hunk 3]
```diff
@@ -10,19 +9,22 @@ Using these abstractions, Pekko also provides libraries for persistence, streams
...
 - Make sure you have installed a Java Development Kit (JDK) version 11 or later.
 - Make sure you have [sbt](https://www.scala-sbt.org/) installed and using this JDK.
 - Make sure you also have Java 8 installed (JDK or JRE).
-- Use git to clone the [repo](https://github.com/apache/incubator-pekko) or download a source release from https://pekko.apache.org
-- Open a command window and change directory to the directory where you installed the source
+- [Graphvis](https://graphviz.gitlab.io/download/) is needed for the scaladoc generation build task, which is part of the release.
+
+### Running the Build
+- Open a command window and change directory to your preferred base directory
+- Use git to clone the [repo](https://github.com/apache/incubator-pekko) or download a source release from https://pekko.apache.org (and unzip or untar it, as appropriate)
```
Comment (raboof) [inline README.md:26]: (offtopic: I wonder if we need to do anything special to make sure the version number is determined correctly when building from a zip/tar)

**Review by raboof — APPROVED** [review]
LGTM except for two typo's, nice with the slight restructure/reformat

File: README.md (hunk 3 above)
Comment (pjfanning) [reply README.md:26]: interesting point - let me experiment with that

Comment (pjfanning) [reply README.md:26]: @raboof I ran a build in a dir with the source but that was not under git control. The jars that were built came out with names like:

pekko-remote_2.13-HEAD+20230524-1416.jar

Comment (pjfanning) [reply README.md:26]: HEAD and the date and time

**Pull request merged**